scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -279,21 +308,22 @@ object Measurer {
    *  susceptible to actual randomness, because adding artificial noise increases
    *  the confidence intervals.
    */
-  trait Noise extends Measurer {
+  trait Noise extends Measurer[Double] {
```

Comment:
I guess that would be one way to do it. I don't think it's a high priority, though. It's possible that we should eventually deprecate `Noise` measurer altogether.
And I'd still avoid the `Runner`/`Gauge` split.